cmd/kyma/install/cmd.go

```diff
@@ -70,8 +70,9 @@ Both installation types continue with the following steps:
    5. Set the admin password.
    6. Patch the Minikube IP.
    `,
-		RunE:    func(_ *cobra.Command, _ []string) error { return cmd.Run() },
-		Aliases: []string{"i"},
+		RunE:       func(_ *cobra.Command, _ []string) error { return cmd.Run() },
+		Aliases:    []string{"i"},
+		Deprecated: "install is deprecated!",
```

Comment:
The CLI output is: "Command "install" is deprecated, install is deprecated!" which is redundant. I would rather add a comment to instead use the new deploy command

Reply:
good idea will do it